cypress/integration/version/task_filters.ts

```diff
@@ -183,7 +183,11 @@ describe("Tasks filters", () => {
   describe("Task Base Statuses select", () => {
     const urlParam = "baseStatuses";
     before(() => {
+      cy.intercept("POST", GQL_URL, (req) => {
+        aliasQuery(req, "PatchTasks");
+      });
       cy.visit(pathTasks);
+      cy.wait("@gqlPatchTasksQuery");
```

Comment:
I think it's kind of strange how we don't do this for the other filters we test on this page. Should we?

Reply:
cypress was having issues toggling the table filter so the lines above make sure the table data is loaded. I don't think we need to do this for the other filters since they didn't have the same flake condition